drivers/hmis/lib/hmis_errors/api_error.rb

```diff
@@ -8,6 +8,7 @@ module HmisErrors
   class ApiError < StandardError
     attr_reader :display_message
     INTERNAL_ERROR_DISPLAY_MESSAGE = 'An error occurred'.freeze
+    STALE_OBJECT_ERROR = "We couldn't save your changes because someone else made updates to the same data while you were working on it. Please refresh the page and try saving your changes again. If the issue persists, contact support for assistance.".freeze
```

Comment:
"we" feels slightly off to me? but dont feel strongly about it
```suggestion
    STALE_OBJECT_ERROR = "Your changes couldn't be saved because someone else made updates to the same data while you were working on it. Please refresh the page and try saving your changes again. If the issue persists, contact support for assistance.".freeze
```